splinepy/helpme/extract.py

```diff
@@ -585,3 +608,15 @@ def spline(self, splitting_plane=None, interval=None):
             return spline_copy
         else:
             return spline(self._spline, splitting_plane, interval)
+
+
+# Use function docstrings in Extractor functions
+Extractor.edges.__doc__ = edges.__doc__
```

Comment:
I think you can integrate this inside the function

Reply:
setting it within a function is not possible because the lines are not evaluated. within the class body also does not work, because you can not access these names as class variables. This is why I put them outside

Reply:
```python
def c():
    """
    I am c
    """
    pass

class A:
    def b():
        pass

    b.__doc__ = c.__doc__

print(c.__doc__)
print(A.b.__doc__)
```